### PR TITLE
chore: limit metrics collection

### PR DESCRIPTION
**Pull request by ktechmidas**
## Issue being fixed or feature implemented
We're collecting everything and sending to AWS which is causing an uptick in costs

## What was done?
Limit metrics collection to just CPU for now

## How Has This Been Tested?
testnet

## Breaking Changes
None

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

File: ansible/deploy.yml [hunk 1]
```diff
@@ -61,7 +61,44 @@
...
+            append_dimensions:
+              ImageId: "${aws:ImageId}"
+              InstanceId: "${aws:InstanceId}"
+              InstanceType: "${aws:InstanceType}"
+              AutoScalingGroupName: "${aws:AutoScalingGroupName}"
+            aggregation_dimensions:
+              - - AutoScalingGroupName
+              - - InstanceId
+                - InstanceType
+              - []
```
Comment (strophy) [inline ansible/deploy.yml:100]: typo here?

**Comment by strophy** [comment]
Thanks for working on this so quickly. Is it possible to load the config from a file somewhere instead of pasting it directly into `deploy.yml`?

File: ansible/deploy.yml (hunk 1 above)
Comment (ktechmidas) [reply ansible/deploy.yml:100]: Nope: https://github.com/christiangda/ansible-role-amazon-cloudwatch-agent

unless upstream has a typo

**Comment by ktechmidas** [comment]
Yeah let me just make sure it works first... then we'll clean it up, this PR is just to ensure people don't run it over again :)

File: ansible/deploy.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@
       when: openvpn_enabled
       tags: vpn
 
-- name: Setup jq, Python and Docker
+- name: Setup jq, Python, cwagent and Docker
```
Comment (strophy) [inline ansible/deploy.yml:38]:
```suggestion
- name: Setup jq, Python, CWAgent and Docker
```

File: ansible/roles/cwagent/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+- name: Set up cloudwatch
```
Comment (strophy) [inline ansible/roles/cwagent/tasks/main.yml:2]:
```suggestion

- name: Set up CloudWatch
```

**Review by strophy — APPROVED** [review]

**Pull request merged**